seed/ts-sdk/bytes-upload/README.md

```diff
@@ -0,0 +1,196 @@
+# Seed TypeScript Library
+
+[![fern shield](https://img.shields.io/badge/%F0%9F%8C%BF-Built%20with%20Fern-brightgreen)](https://buildwithfern.com?utm_source=github&utm_medium=github&utm_campaign=readme&utm_source=Seed%2FTypeScript)
+[![npm shield](https://img.shields.io/npm/v/@fern/bytes-upload)](https://www.npmjs.com/package/@fern/bytes-upload)
+
+The Seed TypeScript library provides convenient access to the Seed API from TypeScript.
+
+## Installation
+
+```sh
+npm i -s @fern/bytes-upload
+```
+
+## Reference
+
+A full reference for this library is available [here](./reference.md).
+
+## Usage
+
+Instantiate and use the client with the following:
+
+```typescript
+import { createReadStream } from "fs";
+import { SeedBytesUploadClient } from "@fern/bytes-upload";
+
+const client = new SeedBytesUploadClient({ environment: "YOUR_BASE_URL" });
+await client.service.upload(createReadStream("path/to/file"));
+```
+
+## Exception Handling
+
+When the API returns a non-success status code (4xx or 5xx response), a subclass of the following error
+will be thrown.
+
+```typescript
+import { SeedBytesUploadError } from "@fern/bytes-upload";
+
+try {
+    await client.service.upload(...);
+} catch (err) {
+    if (err instanceof SeedBytesUploadError) {
+        console.log(err.statusCode);
+        console.log(err.message);
+        console.log(err.body);
+        console.log(err.rawResponse);
+    }
+}
+```
+
+## File Upload
+
+You can upload files to a file upload endpoint like this:
```

Comment:
The SDK accepts a variety of types for file upload parameters: 
- Stream types: `fs.ReadStream`, `stream.Readable`, and `ReadableStream`
- Buffered types: `Buffer`, `Blob`, `File`, `ArrayBuffer`, `ArrayBufferView`, and `Uint8Array`